Matlab_Scripts/5_Depth_Colourless/Pilot/master.m

```diff
@@ -0,0 +1,17 @@
+% This script either runs preprocesses or analysis, for the appropriate
+% condition & event type depending on its settings
+% change preprocessing parameters here
+Wrapper_VR_P3_Depth_Colourless_Pilot
```

Comment:
ok, good to be flexible, but ultimately we want a reproducible analysis pipeline that is linear and that needs to input from the user. so we need to separate these scripts out or call this function with different settings from the master file

we want to have the goal of publishing the study, and the data, and analysis files, and all the user has to do is indicate the data path and run this file in order to reproduce our analysis